src/editor/panel.ts

```diff
@@ -65,29 +36,36 @@ export default class EditorPanel extends Widget {
     this._notebookConfig = options.notebookConfig;
 
     this._context.model.stateChanged.connect(() => {
-      this._stateChanged();
+      this._checkMetadata();
     });
 
-    this.addClass('grid-panel');
+    this._context.model.contentChanged.connect(() => {
```

Comment:
When closing the voila-editor and continuing working with the notebook this signals still connected.
We need to disconnect but I'm not sure how to do it.
https://github.com/hbcarlos/voila-editor/issues/21

Reply:
A common pattern for disconnecting signals when widgets are destroyed is to do it in the `dispose` method.

Using:

```typescript
Signal.clearData(this).
```

Or manually disconnecting the signals with `.disconnect()`